### ISSUE TITLE
Features: Distributed computation

### ISSUE THREAD

**Issue by arita37** [mattja/distob#2]
Hello,

Find your project interresting.
Just wondering if there is a way to generate many instances of a class:  c1, c2,...cn (all having different parameters) and send them on the clusters and do individual compute ( c1.compute() ) and get back the results into in the local computer ?

Thanks

**Comment by mattja** [comment]
Yes, you can use the `__init__` function of your class to give different parameters to each instance. 
For example, here giving each instance slightly different random values:
```python
import distob
import numpy as np

class C(object):
    def __init__(self):
        self.a = np.random.normal(7.0, 2.0)

    def compute(self, x):
        return 100*x + self.a

N = 10
instances = [C() for i in range(N)]
instances = distob.scatter(instances)

results = distob.call_all(instances, 'compute', 1.0)
print(results)
```
The function calls to compute() will be executed in parallel on the cluster.
